app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  before_action :signed_in_user, only: %i[new create]
```

Comment:
Please for **only**, could you use symbols instead of an array of string. Something like `:new` ?

Reply:
Fixed it, thanks

Reply:
You should do something like this: `only: [:new, :create]`. I don't think this is a blocker but that is what the TSE is trying to say.